packages/showcases/css/stories/components/quantity/examples/overview.html

```diff
@@ -0,0 +1,116 @@
+<style>
+  .block {
+    margin: 20px;
+  }
+  body {
+    background-color: var(--vtmn-semantic-color_background-tertiary);
+  }
+</style>
+<div class="block">
+  <div class="vtmn-quantity">
```

Comment:
`.vtmn-quantity-container` should be the parent of `.vtmn-quantity` and not the inverse.
See text input or select :-)